docs/integrations/aws-lambda.md

```diff
@@ -0,0 +1,36 @@
+# AWS Lambda
+
+The [`logfire.instrument_aws_lambda`][logfire.Logfire.instrument_aws_lambda] function can be used to
+instrument AWS Lambda functions to automatically send traces to **Logfire**.
+
+## Installation
+
+Install `logfire` with the `aws-lambda` extra:
+
+{{ install_logfire(extras=['aws-lambda']) }}
+
+## Usage
+
+To instrument an AWS Lambda function, call the `logfire.instrument_aws_lambda` function after defining
+the handler function:
+
+```python
+import logfire
+
+logfire.configure()  # (1)!
+
+
+def handler(event, context):
+    return 'Hello from Lambda'
+
+logfire.instrument_aws_lambda()  # (2)!
+```
+
+1. Remember to set the `LOGFIRE_TOKEN` environment variable on your Lambda function configuration.
+2. The `logfire.instrument_aws_lambda` function must be called after defining the handler function.
```

Comment:
I think this should be enforced with code, e.g. require calling `logfire.instrument_aws_lambda(handler)` or check `globals()['handler']`.

Reply:
I tried to do it with `globals()`, but the function was not there, I guess I needed to go up the stack and get the locals up? It was easier to add the `lambda_handler`.